tools/pipelines/templates/build-npm-package.yml

```diff
@@ -11,6 +11,10 @@ parameters:
   type: string
   default: ci:build
 
+- name: taskGenerateRollups
+  type: string
+  default: build:docs
```

Comment:
docs are already built as part of the main build step, so I don't think you need this.

Reply:
While I was testing, the rollups were not generated. so had to add this additional step. can you point me to the code where we first run `build:docs` and then `build`?

Reply:
You can see it in the build logs: https://dev.azure.com/fluidframework/public/_build/results?buildId=198697&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=bfe27d5f-9219-57c0-0d67-1b2946a32921 Look for api-extractor in the output. The rollups are not generated because none of the packages have rollups enabled in their api-extractor config. until that's done they won't generate the rollups.

Reply:
> The rollups are not generated because none of the packages have rollups enabled in their api-extractor config. until that's done they won't generate the rollups.

I think #17693 will pave the way for this. @Josmithr will there need to be a follow-up PR to enable rollups for all packages?